### PR TITLE
Add uGAS-0921 price identifier

### PR DESCRIPTION
**Pull request by rossgalloway**
New UMIP for uGAS-0921 

The price feed implementation in UMIP-22 is for Uniswap. We will be pooling liquidity on Sushiswap. Is there a different implementation for Sushiswap and do we need to specify it?

### CODE REVIEW

File: UMIPs/GASETH-0921price identifier.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| Status     | draft                                                                                                                                 |
+| Created    | June 15th, 2021                                                                                                                              |
+| Link to Discourse| https://discourse.umaproject.org/t/add-ugas-0921-price-identifier/1196
+
+## SUMMARY
+This UMIP will reference a synthetic token to be created with this price identifier. This token will be referred to as 'uGAS' and will represent the token that tracks this identifier with the most ETH volume on Sushiswap unless a different contract is determined by voters to be more legitimate.
+
+This follows the exact same process as UMIP-22 but uses a different timestamp.
+
+The DVM should support requests for a price that resolves to either the median monthly Ethereum gas price or a 2-hour Time-Weighted Average Price (TWAP) on the highest volume Uniswap or sushiswap ETH/uGAS pool. The price resolution method to use will depend on the the timestamp the price request was made at.
```
Comment (Reinis-FRP) [inline UMIPs/GASETH-0921price identifier.md:15]: 1st paragraph of Summary only refers to Sushiswap - should it be changed to Uniswap or Sushiswap everywhere?

File: UMIPs/GASETH-0921price identifier.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| UMIP Title | Add GASETH-0921 as a supported price identifier                                                                                             |
+| Authors    | Ross (ross@yam.finance)
+| Status     | draft                                                                                                                                 |
+| Created    | June 15th, 2021                                                                                                                              |
+| Link to Discourse| https://discourse.umaproject.org/t/add-ugas-0921-price-identifier/1196
+
+## SUMMARY
+This UMIP will reference a synthetic token to be created with this price identifier. This token will be referred to as 'uGAS' and will represent the token that tracks this identifier with the most ETH volume on Sushiswap unless a different contract is determined by voters to be more legitimate.
+
+This follows the exact same process as UMIP-22 but uses a different timestamp.
```
Comment (Reinis-FRP) [inline UMIPs/GASETH-0921price identifier.md:13]: If it is expected to update this on a quarterly basis, maybe it is worth considering creating a generalized uGAS price identifier where relevant parameters can be passed as ancillary data:
- expiration timestamp - to be used by FPL price identifier switcher
- AMM pool address - for pre-expiration

Comment (rossgalloway) [reply UMIPs/GASETH-0921price identifier.md:13]: Yes, see comment in uStonks UMIP

File: UMIPs/GASETH-0921price identifier.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+# MARKETS & DATA SOURCES
+
+Please refer to [umip-22](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-22.md).
+
+
+# PRICE FEED IMPLEMENTATION
+
+To further explain the price feed implementation beyond what is stated in [umip-22]: The price feed being used is the Uniswap price feed and only the Uniswap TWAP calculation will need to be queried in real-time. The Uniswap price feed is referenced [here](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js).
```
Comment (rossgalloway) [inline UMIPs/GASETH-0921price identifier.md:33]: @Reinis-FRP and @smb2796 Is it fine that this refers to the uniswap price feed implementation? Is there a different one for sushiswap?

File: UMIPs/GASETH-0921price identifier.md (hunk 1 above)
Comment (rossgalloway) [reply UMIPs/GASETH-0921price identifier.md]: updated to remove Uniswap

File: UMIPs/GASETH-0921price identifier.md (hunk 3 above)
Comment (smb2796) [reply UMIPs/GASETH-0921price identifier.md:33]: Same one is used for sushiswap 👍 , so this is correct.

**Review by therealhexi — APPROVED** [review]
Looks good to community reviewer

**Review by smb2796 — APPROVED** [review]
Looks good. Merging in and moving to Last Call

**Review by CarelessDoge — APPROVED** [review]

**Pull request merged**